Quickstart.md

```diff
@@ -0,0 +1,197 @@
+# QuickStart guide on how to setup AppCli with a sample application
+
+### Prerequisites
+
+- python
+- docker
+- docker-compose
+
+
```

Comment:
One empty line, not two. Don't add unnecessary spacing

Reply:
To add to this - a markdown linter in vscode would highlight these issues for you as you go